Demo/Application/Features/Preferred Payment Methods/BraintreeDemoPreferredPaymentMethodsViewController.swift

```diff
@@ -0,0 +1,132 @@
+import UIKit
+
+class BraintreeDemoPreferredPaymentMethodsViewController: BraintreeDemoBaseViewController {
```

Comment:
Ah, I see if we use `PaymentButtonBaseViewController` I think we will get access to a lot of helpers here for setting up buttons/the view/etc